utils.go

```diff
@@ -130,10 +143,6 @@ func calculateBridgeIP(n *NetConf) (*net.IPNet, error) {
 		err         error
 	)
 
-	if n.BrSubnet == "" {
-		return nil, fmt.Errorf("mandatory bridgeSubnet not specified in config")
-	}
-
 	_, brNetworkIPNet, err := net.ParseCIDR(n.BrSubnet)
```

Comment:
Also, net.ParseCIDR(n.BrSubnet) will return an error if BrSubnet is unset, so we are still protected from future incorrect usages of this function.